src/components/Button/index.js

```diff
@@ -8,24 +8,31 @@ const Button = ({
   children,
   className,
   disabled,
-  onClick,
+  emphasized,
+  href,
   id,
+  onClick,
+  ...attributes
 } = {}) => {
-  let classes = classNames('resin-control', className);
+  let classes = classNames('resin-control', 'resin-control-btn', className, { emphasized });
 
   if (btnStyle) {
-    classes += ` resin-control_btn-${btnStyle}`;
+    classes += ` resin-control-btn-${btnStyle}`;
   }
 
+  const Tag = href ? 'a' : 'button';
```

Comment:
This reminds me we should have a Link primitive. That basically checks if a link is for an external site or an internal one and then does html5 push state if it's internal and regular <a> if not. We can chat on it tomorrow.